### ISSUE TITLE
Notify maintainer when a repo master build has failed for a prolonged period

### ISSUE THREAD

**Issue by jameswnl** [ManageIQ/miq_bot#367]
For a less popular repo, the master build could be in failed state for prolonged period without being noticed. 

E.g. [a 25 day failure here](https://travis-ci.org/ManageIQ/manageiq-providers-foreman/builds). This build failure is caused by a core change and we should have caught and fixed this problem early on vs now being notified by a bug report.

So a feature request: miq-bot to notify (e.g. by emailing) maintainers after say `failure not fixed within 1 day`.

**Comment by Fryguy** [comment]
Travis has a lot of features built in for notifications

- email: https://docs.travis-ci.com/user/notifications/#Configuring-email-notifications
- gitter (via irc): https://docs.travis-ci.com/user/notifications/#Configuring-IRC-notifications
- or both

I would prefer one of those over coding our own home grown notification

**Comment by jameswnl** [comment]
yeah, that'd would work. I'll keep this issue open until we set the other things up.
